src/main/java/jalau/at18/architects/ScienceComparator.java

```diff
@@ -0,0 +1,13 @@
+package jalau.at18.architects;
+
+public class ScienceComparator implements Comparator {
+    private ScienceCounter scienceCounter;
+
+    public ScienceComparator(ScienceCounter scienceCounter) {
+        this.scienceCounter = scienceCounter;
+    }
+
+    public boolean compare(Card cardOne, Card cardTwo) {
+        return scienceCounter.traverseTheScienceElementsList();
```

Comment:
Something that looks a little bit weird is that current logic could be simpler without `ScienceCounter` and it would work the same. Say
```java
return cardOne == cardTwo
```
In other words, what would be the purpose of scienceCounter? it does not seem to be counting anything and just returns the first and last elements of its list, which at the end are cards one and two.